ext/standard/tests/math/number_format_negative_zero.phpt

```diff
@@ -4,13 +4,28 @@ Prevent number_format from returning negative zero
 <?php
 
 $number = -1.15E-15;
+var_dump($number);
```

Comment:
I would prefer a new test for this as this one is for negative number and not negative decimal. You should probably also test it with positive numbers.